### ISSUE TITLE
Bug: named workspaces and focusmon

### ISSUE THREAD

**Issue by fedoranvar** [vaxerski/Hyprland#112]
Good day to you!

I have two monitors ( left and right) and this keymaps in config:

 ```bash
 bind=SUPERSHIFT,H,focusmonitor,l
bind=SUPERSHIFT,J,focusmonitor,d
bind=SUPERSHIFT,K,focusmonitor,u
bind=SUPERSHIFT,L,focusmonitor,r

... 

 bind=SUPER,Q,workspace,name:term
bind=SUPER,W,workspace,name:web
bind=SUPER,E,workspace,name:chat
```

when i fill my monitors with named workspaces , i can `focusmonitor` to right , but can't to left

BUT

if i replace named workspaces with simple numbered (1,2,3), then all works well.
```bash
bind=SUPER,Q,workspace,1
bind=SUPER,W,workspace,2
bind=SUPER,E,workspace,3
```

**Comment by vaxerski** [comment]
fixed with e4d494aed4494b31adfe893453d1644a6ccb6439 I think. Please verify.

**Comment by fedoranvar** [comment]
yes. confirm that bug is fixed.

thank you!
